monkey/monkey_island/cc/resources/log.py

```diff
@@ -1,35 +0,0 @@
-import json
-
-from bson import ObjectId
-from flask import request
-
-from monkey_island.cc.database import mongo
-from monkey_island.cc.resources.AbstractResource import AbstractResource
-from monkey_island.cc.resources.request_authentication import jwt_required
-from monkey_island.cc.services.log import LogService
```

Comment:
This Log resource is actually the sole user of `LogService`, so we should remove `LogService` as well

Reply:
Thanks @cakekoa